### PR TITLE
feat: add options in flat run

### PR DESCRIPTION
**Pull request by keita-determined**
## Ticket

[ET-229]

## Description

Add table settings menu, and removed dead code (infinite scroll is removed permanently)

## Test Plan

- Enable Flat Run table
- Check if `row height` changes by changing option settings and the settings should persist

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[ET-229]: https://hpe-aiatscale.atlassian.net/browse/ET-229?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/components/OptionsMenu.tsx [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
 import Icon from 'hew/Icon';
-// import Toggle from 'hew/Toggle';
-import { literal, TypeOf, union } from 'io-ts';
+import { TypeOf } from 'io-ts';
 import { useMemo } from 'react';
 
 import { valueof } from 'utils/valueof';
 
-// import css from './OptionsMenu.module.scss';
-
```
Comment (johnkim-det) [inline webui/react/src/components/OptionsMenu.tsx:11]: also delete file?

**Review by johnkim-det — APPROVED** [review]
one comment but otherwise LGTM, probably makes sense to update https://hpe-aiatscale.atlassian.net/browse/ET-259 since you addressed it here?

File: webui/react/src/components/OptionsMenu.tsx (hunk 1 above)
Comment (keita-determined) [reply webui/react/src/components/OptionsMenu.tsx:11]: good catch, removed it

**Comment by keita-determined** [comment]
> one comment but otherwise LGTM, probably makes sense to update https://hpe-aiatscale.atlassian.net/browse/ET-259 since you addressed it here?

i'll update the ticket. i think it should be still `TODO` because i didn't refactor code.

**Pull request merged**